.travis.yml

```diff
@@ -17,9 +17,12 @@ before_script:
   - "[ ! -f wrk/bin/wrk ] && rm -rf wrk && git clone https://github.com/wg/wrk.git && make -C wrk && mkdir wrk/bin && mv wrk/wrk wrk/bin || true"
   - export PATH=$PATH:$PWD/wrk/bin/
 script:
-  - make lint
-  - make test-travis
-  - make bench
+  - npm run lint
+  - jest --coverage
```

Comment:
I'm not sure that would be very helpful, I think `npm test --coverage` is good enough.